sklearn/preprocessing/imputation.py

```diff
@@ -357,6 +379,18 @@ def transform(self, X):
 
             X.data[mask] = astype(valid_statistics[indexes], X.dtype,
                                   copy=False)
+
+            if self.add_missing_indicator:
+                mask_matrix = X.__class__((mask, X.indices.copy(),
+                                           X.indptr.copy()), shape=X.shape, dtype=X.dtype)
```

Comment:
PEP8. Please run all your code through `flake8` (but only fix mistakes you've created)